.cirrus.yml

```
@@ -62,10 +56,30 @@ build_task:
   upload_caches: [ "cargo", "targets", "bin" ]
 
 
+build_aarch64_task:
+  alias: "build_aarch64"
+  # Compiling is very CPU intensive, make it chooch quicker for this task only
+  ec2_instance: &standard_build_ec2_aarch64
+    image: ami-0f89dfa699fe590b0
+    type: t4g.xlarge
+    region: us-east-1
+    architecture: arm64  # CAUTION: This has to be "arm64", not aarch64.
+  cargo_cache: *cargo_cache
```

This task is going to cause cache-clashes with the build task.  This will cause a race in dependent tasks over which cache to use (since the key is the same).  So we need a unique cache key for the non-x86_64 tasks to prevent this.

```suggestion
  cargo_cache: &cargo_cache_aarch64
      <<: *cargo_cache
      fingerprint_key: "cargo_v1_${DEST_BRANCH}_aarch64"
```

---

Note: This applies to ALL the `aarch64` cache-related anchors and aliases :cry: